elex/api/utils.py

```diff
@@ -98,7 +98,6 @@ def api_request(path, **params):
     params = sorted(params.items())  # Sort for consistent caching
 
     url = '{0}{1}'.format(elex.BASE_URL, path.replace('//', '/'))
-
```

Comment:
No need to make this change to a file you aren't otherwise editing; think this is probably here because of a debug print we were doing earlier.